about/structure.md

```diff
@@ -0,0 +1,60 @@
+# Structure and governance of 2i2c
+
+2i2c is an initiative dedicated to supporting open infrastructure in interactive computing for research and education, as well as the open source communities that underlie this infrastructure. Its host organization is {ref}`structure:icsi`.
+
+This page describes the major organizational structures of 2i2c, and how they relate to governance and operations.
+
+:::{admonition} A few notes on governance
+- All 2i2c members act as individuals, and not on behalf of their affiliated institutions.
+- All decisions made by the 2i2c Steering Council or Teams must abide by the policies of its host organization, {ref}`structure:icsi`.
+:::
+
+## Steering Council
+
+The Steering Council defines the mission, vision, and values of 2i2c. It also sets the strategic direction and priorities for 2i2c. The Steering Council provides oversight to the Executive Director of 2i2c and the Operations Team.
+
+The Steering Council makes decisions via consensus, and is restricted by the legal obligations and policies of ICSI. It uses the following process for these decisions:
+
+* Open an issue in the `meta/` GitHub repository using [the {guilabel}`decision` template](https://github.com/2i2c-org/meta/issues/new?assignees=&labels=decision&template=decision.md&title=%7B%7B+Decision+Summary+%7D%7D).
+* Have conversation around the decision in that issue. This may happen in the issue directly, or in ancillary issues, meetings, etc that are relevant. The decision issue should link to these relevant spaces.
+* Once everybody has voted, note the decision in the issue and close it.
```

Comment:
To my understanding, the notions of "voting" and "consensus" (above) are incompatible.

Reply:
Ah - I was interpreting consensus to mean "every vote must be in favor in order to proceed". Is that not clear here? Or does it differ from your understanding of consensus?

Reply:
It was not clear to me.

Reply:
I'll make that language more explicit 👍